doc/tutorials/content/sources/qt_visualizer/pclviewer.cpp

```diff
@@ -84,14 +105,22 @@ PCLViewer::RGBsliderReleased ()
     point.b = blue;
   }
   viewer->updatePointCloud (cloud, "cloud");
-  ui->qvtkWidget->update ();
+#if VTK_MAJOR_VERSION > 8
```

Comment:
Make this a function maybe.

PS: this is a tutorial. Please check for code line number updates